config/game_options.txt

```diff
@@ -1,7 +1,7 @@
 ## HEALTH ###
 
 ##Damage multiplier, effects both weapons and healing on all mobs. For example, 1.25 would result in 25% higher damage.
-DAMAGE_MULTIPLIER 1
+DAMAGE_MULTIPLIER 1.25
```

Comment:
woe there

Reply:
Why the hell is this a thing? Change this Rory.

Reply:
Didn't even notice it.

Reply:
pointless change